source/CMakeLists.txt

```diff
@@ -41,12 +41,29 @@ if(UVISOR_PRESENT)
     )
 else()
     message(WARNING
-"\n         *******************************************************************
-         * WARNING (uvisor-lib): unsupported platform; your code will still
-         *                       work but no security feature is provided;
-         *                       UVISOR_DISABLED is set by default
-         *******************************************************************")
+"\n         *********************************************************************
+         * WARNING (uvisor-lib): unsupported platform; your code will still  *
+         *                       work but no security feature is provided;   *
+         *                       UVISOR_DISABLED is set by default           *
+         *********************************************************************")
```

Comment:
Make this change in a separate commit, as it has a different purpose from the "Throw error at inconsistent UVISOR_PRESENT settings" commit. If there are any other warnings without "*" borders, they could also be fixed in this new commit.